src/scripts/browserDeviceProfile.js

```diff
@@ -233,8 +239,20 @@ function supportsHlg(options) {
 }
 
 function supportsDolbyVision(options) {
+    // Safari 13.1 added support for Dolby Vision:
+    //  * https://developer.apple.com/documentation/safari-release-notes/safari-13_1-release_notes
+    const safariSupportsDoVi = browser.safari && (false // eslint-disable-line sonarjs/no-redundant-boolean
```

Comment:
Unclear why Sonarlint doesn't pick up this lint suppression. This is following the same code-style as the rest of the file.